sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -588,9 +588,15 @@ def run_pipeline(self, pipeline, options):
     return result
 
   def _maybe_add_unified_worker_missing_options(self, options):
+    debug_options = options.view_as(DebugOptions)
+    # Streaming is always portable, default to runner v2.
+    if options.view_as(StandardOptions).streaming:
+      if not debug_options.lookup_experiment('disable_runner_v2'):
+        debug_options.add_experiment('beam_fn_api')
+        debug_options.add_experiment('use_runner_v2')
```

Comment:
This will move the runner_v2 selection from Service to SDK and will limit out capability of auto optin/not option to UW if we ever have to.

Is there a specific reason why we want to move runner selection to SDK.